internal/common/params_file.bzl

```diff
@@ -70,25 +98,41 @@ def params_file(
         name,
         out,
         args = [],
+        data = [],
         newline = "auto",
         **kwargs):
     """Generates a UTF-8 encoded params file from a list of arguments.
 
-    Handles $(location) expansions for arguments.
+    Handles variable substitutions for args.
 
     Args:
-      name: Name of the rule.
-      out: Path of the output file, relative to this package.
-      args: Arguments to concatenate into a params file.
-          Subject to $(location) substitutions
-      newline: one of ["auto", "unix", "windows"]: line endings to use. "auto"
-          for platform-determined, "unix" for LF, and "windows" for CRLF.
-      **kwargs: further keyword arguments, e.g. <code>visibility</code>
+        name: Name of the rule.
+        out: Path of the output file, relative to this package.
+        args: Arguments to concatenate into a params file.
+
+        1. Subject to $(location) substitutions.
```

Comment:
this is just copy-pasta from somewhere right? if we can't share it then maybe a comment pointing to the canonical version of it so we have a chance to prevent skew

Reply:
thanks. It's not copy-pasta but similar to another docstring so I'll just reference that. And I'll remove the duplicate docstrings from _ATTRS and so only the macro has the strings.